contents/blog/customer-support-at-posthog.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-05-17
+title: How We Do Customer Support At Our Open-Source DevTool Company
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Mohammed Shehu](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+Customer support is a critical function of any startup, especially if you offer a highly technical product like we do. [Charles Cook](https://www.linkedin.com/in/charlescook1/), Business Head at PostHog, shared his thoughts on how CS was carried out at his previous companies (VITL and Roli) and how customer support has evolved here at PostHog over the past few months. 
+
+[Roli](http://www.roli.com/) sells musical instruments and production software. Because the company deals with creators, the customer support team consists of qualified technical musicians, producers, and composers in their own right. Reflecting the makeup of your customer base is key to ensuring your CS team can relate to the problems that your customers face and provide better, more effective solutions.
+
+[VITL](https://vitl.com/) was a similar story: the company provides personalized nutrition, tailor-made supplements, and at-home nutrition tests to its customers. During Charles' time there, the team was more hybrid, with some of the CS staffers spending 100% of their time on customer support and others splitting their time between support and working on nutrition. Again, this meant that the support team consisted of qualified professionals who could nail down problems and serve up credible solutions.
+
+## Customer support at PostHog
+
+Early-stage startups are typically light on people and heavy on bugs, which means the founding devs handle customer support queries alongside their daily development duties. This was also the case at PostHog, and GitHub was a major source of communication between the team and our users. There also wasn't much structure in the beginning.
+
+Our customers are engineers, and engineers want to talk to and get solutions from other engineers. We also discovered that having our engineers involved in customer support not only turned up solutions for our users faster but also vastly improved the product development process. Contrast this to most devtool startups that go down the 'dedicated, separate customer support team' route too early, which disconnects what you're building from what the user actually needs.
+
+We also use an open-source product support tool called [Paper Cups](https://papercups.io/) that's open-source, lightweight, and packed with tons of useful features. We've integrated it into Slack and email so that we get notifications whenever any new issues pop up and can respond directly in Slack. We've also instituted a Support Hero system that changes every two weeks, with one member of our dev team taking over support queries during each run.
```

Comment:
We could give more detail on how this works - perhaps a small flow chart or something